### DISCUSSION TITLE
Download full article text for offline use

### DISCUSSION THREAD

**Discussion by amit3s** [jocmp/capyreader#476] [Feature Requests]
Hello, 

Please add a fully offline mode by scheduling feed refreshes asking with full article text when online.

**Comment by jocmp** [comment]
Hey @amit3s - is your idea that any new article is saved or would the app give an option to save articles offline one-by-one?

**Reply by crazygolem** [reply]
I for one would like the option to fetch all new articles' whole content when the app refreshes the feeds, and save them for offline reading.

I would enable this option globally if available, but I've seen another RSS reader allowing to configure it for each feed individually.

**Reply by jocmp** [reply]
@crazygolem do you remember any of those RSS readers off the top of your head? I'll take a look at how they work.

**Reply by amit3s** [reply]
> Hey @amit3s - is your idea that any new article is saved or would the app give an option to save articles offline one-by-one?

I'd like to have all articles downloaded offline when the feed refreshes.

HandyNewsReader does this - but also, generally, all RSS feeds worked this way years back.

**Reply by jocmp** [reply]
Fair enough. I'll take a look at HandyNewsReader!

**Reply by crazygolem** [reply]
Flym and Reader You have a per-feed option
